.github/workflows/test_pr.yml

```
@@ -1,53 +1,50 @@
 name: Tests
 
 on:
-    push:
-        branches:
-            - main
-    pull_request:
-        branches:
-            - main
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
 
 env:
-    BUILD_TYPE: Release
+  BUILD_TYPE: Release
 
 jobs:
-    build:
-        name: ${{ matrix.platform }}
-        runs-on: ${{ matrix.platform }}
-        timeout-minutes: 10
-        strategy:
-            fail-fast: false
-            matrix:
-                platform: ["windows-latest", "ubuntu-latest", "macos-latest"]
-        permissions:
-            actions: write
-
-        steps:
-            - name: Cancel Previous Runs
-              uses: styfle/cancel-workflow-action@0.10.0
-              with:
-                  access_token: ${{ github.token }}
-
-            - uses: actions/checkout@v3
-
-            - name: Install Cmake 3.23
-              run: |
-                if [ "$RUNNER_OS" == "Linux" ]; then
-                  sudo apt-get update
-                  sudo apt-get install cmake-data
-                  wget -O cmake.sh https://cmake.org/files/v3.23/cmake-3.23.2-linux-x86_64.sh
-                  sudo bash cmake.sh --skip-license --exclude-subdir --prefix=/usr/local
-                fi
-              shell: bash
-
-            - name: Configure CMake
-              run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}}
-
-            - name: Build
-              run: cmake --build ${{github.workspace}}/build --config ${{env.BUILD_TYPE}}
-
-            - name: Test
-              working-directory: ${{github.workspace}}/build
-              run: ctest -C ${{env.BUILD_TYPE}} -L anyplatform --output-on-failure
-
+  test:
+    name: ${{ matrix.platform }}
+    runs-on: ${{ matrix.platform }}
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ "windows-latest", "ubuntu-latest", "macos-latest" ]
+    permissions:
+      actions: write
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v3
+
+      - name: Install Cmake 3.23
+        if: ${{ matrix.platform == 'ubuntu-latest' }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install cmake-data
+          wget -O cmake.sh https://cmake.org/files/v3.23/cmake-3.23.2-linux-x86_64.sh
+          sudo bash cmake.sh --skip-license --exclude-subdir --prefix=/usr/local
+        shell: bash
+
+      - name: Configure CMake
+        run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}}
+
+      - name: Build
+        run: cmake --build ${{github.workspace}}/build --config ${{env.BUILD_TYPE}}
+
+      - name: Test
+        working-directory: ${{github.workspace}}/build
+        run: ctest -C ${{env.BUILD_TYPE}} -L acquire-core-libs --output-on-failure

```

Don't need to use a test label here.

```suggestion
        run: ctest -C ${{env.BUILD_TYPE}} --output-on-failure

```